### ISSUE TITLE
Waiting for daemons to come online ...

### ISSUE THREAD

**Issue by konstiantyn** [oliverw/miningcore#1277]
I have geth and miningcore installed on different servers. However, miningcore cannot connect with the geth daemon.
![image](https://user-images.githubusercontent.com/33037366/174433608-14c62755-51f0-45bc-8a21-840872ef382d.png)
{
  "$schema": "https://raw.githubusercontent.com/oliverw/miningcore/master/src/Miningcore/config.schema.json",
  "logging": {
    "level": "info",
    "enableConsoleLog": true,
    "enableConsoleColors": true,
    "logFile": "core.log",
    "apiLogFile": "api.log",
    "logBaseDirectory": "/home/pool/logs",
    "perPoolLogFile": false
  },
  "statistics": {
    "statsUpdateInterval": 60,
    "hashrateCalculationWindow": 30,
    "statsCleanupInterval": 92,
    "statsDBCleanupHistory": 365
  },
  "banning": {
    "manager": "Integrated",
    "banOnJunkReceive": false,
    "banOnInvalidShares": false
  },
  "notifications": {
    "enabled": false,
    "email": {
      "host": "smtp.example.com",
      "port": 587,
      "user": "user",
      "password": "password",
      "fromAddress": "info@yourpool.org",
      "fromName": "support"
    },
    "admin": {
      "enabled": false,
      "emailAddress": "user@example.com",
      "notifyBlockFound": true
    }
  },
  "persistence": {
    "postgres": {
      "host": "127.0.0.1",
      "port": 5432,
      "user": "miningcore",
      "password": "**********",
      "database": "miningcore"
    }
  },
  "equihashMaxThreads": 96,
  "paymentProcessing": {
    "enabled": true,
    "interval": 600,
    "shareRecoveryFile": "recovered-shares.txt"
  },
  "api": {
    "enabled": true,
    "listenAddress": "127.0.0.1",
    "port": 4000,
    "metricsIpWhitelist": [
      "127.0.0.1"
    ],
    "rateLimiting": {
      "disabled": true,
      "rules": [
        {
          "Endpoint": "*",
          "Period": "60s",
          "Limit": 1000000
        }
      ],
      "ipWhitelist": []
    }
  },
  "pools": [
    {
      "id": "eth",
      "enabled": true,
      "coin": "ethereum",
      "address": "0x72d5bA8942e4D991cf2e***********************",
      "rewardRecipients": [
        {
          "type": "op",
          "address": "0x72d5bA8942e4D991cf2e***********************",
          "percentage": 0.1
        }
      ],
      "blockRefreshInterval": 500,
      "jobRebroadcastTimeout": 10,
      "clientConnectionTimeout": 600,

      "banning": {
        "enabled": true,
        "time": 600,
        "invalidPercent": 50,
        "checkThreshold": 50
      },
      "ports": {
        "3092": {
          "name": "ASIC",
          "listenAddress": "0.0.0.0",
          "difficulty": 2.03125,
          "varDiff": {
            "minDiff": 2,
    	    // "maxDelta": 2,
            "maxDiff": null,
            "targetTime": 15,
            "retargetTime": 90,
            "variancePercent": 30
          }
        },
        "4092": {
            "name": "GPU-SMALL",
            "listenAddress": "*",
            "difficulty": 0.05,
             "varDiff": {
               "minDiff": 0.05,
               "maxDiff": null,
               "targetTime": 15,
               "retargetTime": 90,
               "variancePercent": 30
             }
          }
      },
      "chainTypeOverride": "Ethereum",
      "daemons": [
        {
          "host": "184.68.x.x",
          "port": 8545
        }
      ],
      "paymentProcessing": {
        "enabled": true,
        "minimumPayment": 0.025,
        "payoutScheme": "PPLNS",
        "payoutSchemeConfig": {
          "factor": 0.5
        },
      "gas": 21000,
      "maxFeePerGas": 100000000000,
      "keepUncles": false,
      "keepTransactionFees": false

      }
    },
  ]

}

  
How to configure?

**Comment by minershaven** [comment]
In your ETH daemon config file do you have it listening for request from your miningcore server ?

`rpcbind=127.0.0.1
rpcallowip=minincore server ip
server=1
listen=1`

if you dont tell the ETH daemon to expect request from it then it just ignores them all

**Comment by XpoolX** [comment]
> 

JAJAJA

**Comment by gunit666** [comment]
↘️↘️↘️↘️💍💍💍💍💍💍💍💍💍💍💍💍💍💍

**Comment by XpoolX** [comment]
> In your ETH daemon config file do you have it listening for request from your miningcore server ?
> 
> `rpcbind=127.0.0.1` `rpcallowip=minincore server ip` `server=1` `listen=1`
> 
> if you dont tell the ETH daemon to expect request from it then it just ignores them all

This is not how ETH daemon works, stop posting bullshit

**Comment by oliverw** [comment]
> This is not how ETH daemon works, stop posting bullshit

Stay civilized.

**Comment by XpoolX** [comment]
> > This is not how ETH daemon works, stop posting bullshit
> 
> Stay civilized.
Omg, you're right, im very sorry.

**Comment by minershaven** [comment]
You're right. Didn't click in my head that it was ETH. Just finished setting up btg and vtc so had that procedure stuck in the brain.  @oliverw thanks for trying to keep things civil here.

**Comment by oliverw** [comment]
So, is this still an issue or resolved?
